packages/vite-plugin-cloudflare/src/plugins/virtual-modules.ts

```diff
@@ -84,6 +84,12 @@ export function getExportTypes(module) {
 					break;
 				}
 			}
+
+			if (!exportType) {
+				exportType = "DurableObject";
+			}
+		} else if (typeof value === "object" && value !== null) {
+			exportType = "WorkerEntrypoint";
 		}
```

Comment:
If we wanna avoid false postives like Array and make sure it is a plain object:

```suggestion
		} else if (value != null && Object.getPrototypeOf(value) === Object.prototype) {
			exportType = "WorkerEntrypoint";
		}
```